lineapy/editors/ipython.py

```diff
@@ -109,6 +117,12 @@ def input_transformer_post(lines: List[str]) -> List[str]:
 
         STATE = CellsExecutedState(tracer, code=code)
     else:
+        if STATE.tracer.db.url != options.database_connection_string:
+            configure_logging()
+            db = RelationalLineaDB.from_configuration(options)
+            tracer = Tracer(db, SessionType.JUPYTER, session_name, GLOBALS)
+            STATE = CellsExecutedState(tracer, code=code)
+
```

Comment:
<img width="921" alt="Screen Shot 2022-05-31 at 11 40 27 AM" src="https://user-images.githubusercontent.com/504219/171261648-7ae1cbb1-0e0b-4fe9-9367-08b6a74f105a.png">

I'm not sure what your intent is when the database connection changes is, but the above screenshot is a really bad UX. We should at least warn the user when they call the API at the wrong time.